File: KalyanScalaTraining/src/com/orienit/scala/training/oop/StudentApp.scala
package com.orienit.scala.training.oop

object StudentApp {
  def main(args: Array[String]): Unit = {
    val s1 = new Student("kalyan", 1, "spark")
    println("Scala: Student1: " + s1)

    val s2 = new Student("venkat", 2)
    println("Scala: Student2: " + s2)

    val s3 = new Student("raj")
    println("Scala: Student3: " + s3)

    val s11 = new Student(name = "kalyan", id = 1, "spark")
    println("Scala: Student11: " + s11)

    val s12 = new Student(id = 1, name = "kalyan", course = "spark")
    println("Scala: Student12: " + s12)

    val s13 = new Student(name = "kalyan", id = 1)
    println("Scala: Student13: " + s13)

  }
}

File: KalyanScalaTraining/src/com/orienit/java/training/oop/StudentApp.java
package com.orienit.java.training.oop;

public class StudentApp {
	public static void main(String[] args) {
		Student s1 = new Student();
		s1.setName("kalyan");
		s1.setId(1);
		s1.setCourse("spark");
		System.out.println("Java: Student1: " + s1);

		Student s2 = new Student("venkat");
		s2.setId(2);
		s2.setCourse("spark");
		System.out.println("Java: Student2: " + s2);
		
		Student s3 = new Student("venkat", 2);
		s3.setCourse("spark");
		System.out.println("Java: Student3: " + s3);
		
		Student s4 = new Student("venkat", 3, "spark");
		System.out.println("Java: Student4: " + s4);

	}
}












